internal/server/gen/server.pb.go

```diff
@@ -1,13 +1,14 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
+// 	protoc-gen-go v1.25.0
```

Comment:
Nit: should this version number be locked and not changed?

Reply:
Ahhhhh 😭 that was adjusted back on my machine because of https://github.com/hashicorp/waypoint-plugin-sdk/pull/32 😕

Reply:
Fixed it

Reply:
Do we need to update the proto version on the SDK to match what we're using here? That would be my bad -- I should have done both at once when I upped the version.

Reply:
We should, but when we do it's going to break proto gen for anyone on the older version. There's a go import path issue that needs to be fixed (I have the fix ready for when we are ready to bump the version)